treeherder/webapp/api/logslice.py

```diff
@@ -78,7 +84,10 @@ def list(self, request, project, jm):
                     elif i >= end_line:
                         break
 
-                    lines.append({"text": line, "index": i})
+                    if format == 'json':
+                        lines.append({"data": json.loads(line), "index": i})
```

Comment:
Another great point.  Making a change to basically default to text, unless the ``log_name`` is ``mozlog_json`` in which case it will parse the json.  You're right that a param like ``format`` usually has to do with the return value, not the data value.